public/docs/ts/latest/glossary.jade

```diff
@@ -158,10 +156,10 @@ block includes
     to a [View](#view) and handling most of the view’s display
     and user-interaction logic.
 
-    The Component is one of the most important building blocks in the Angular system.
+    The *Component* is one of the most important building blocks in the Angular system.
```

Comment:
Component -> component

(true throughout, unless you're referring to the Component type. the Angular docs have a big case of Over Capitalization.)